05/app.js

```diff
@@ -79,6 +79,7 @@ function checkIsFilled() {
         newLi.innerText = value;
         console.log(messages);
         messages.appendChild(newLi);
+        arr = [];ś
```

Comment:
To nadal nie rozwiązuje problemu.
![Screenshot 2024-04-05 at 08 10 40](https://github.com/devmentor-pl/practice-js-forms/assets/55095257/311f2d61-308a-48eb-adb8-547c0aa54113)